k8s/namespaces/fees-pay/ccpay-payment-app/aat.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ccpay-payment-app
+spec:
+  releaseName: ccpay-payment-api
+  values:
+    java:
+      environment:
+        TRUSTED_S2S_SERVICE_NAMES: "cmc,cmc_claim_store,probate_frontend,divorce_frontend,ccd_gw,bar_api,api_gw,finrem_payment_service,ccpay_bubble,jui_webapp,xui_webapp,fpl_case_service,iac,probate_backend,nfdiv_case_api,civil_service,prl_cos_api,adoption_web,refunds_api"
+        DUMMY_RESTART_VAR: false
```

Comment:
is `DUMMY_RESTART_VAR `needed? default value is false from ccpay-payment-app.yaml

Reply:
https://github.com/hmcts/cnp-flux-config/blob/payment-api-non-prod-v1/k8s/namespaces/fees-pay/ccpay-payment-app/ccpay-payment-app.yaml#L18 reference

Reply:
kept DUMMY_RESTART_VAR  in env specific file and removed from ccpay-payment-app.yam